doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -6,131 +6,389 @@ Cite Test Guide
+
+          git clone https://github.com/geosolutions-it/geoserver.git
+
+   #.  go the cite directory.
+
+       .. code:: shell
+
+          cd geoserver/build/cite
+
+   #.  inside will find a structure like below with a list of directories with the name of the suites to run.
+
+       .. code:: shell
+
+          cite
+          |-- forms
+          |-- geoserver
+          |-- run-test.sh
+          |-- wcs10
+          |-- wcs11
+          |-- wfs10
+          |-- wms11
+          |-- wms13
+          |-- wfs11
+          |-- logs
+          |-- docker-compose.yml
+          |-- postgres
+          |-- README.md
+          `-- Makefile
+
+Running the suite tests.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   There is 2 way to run the suites, one is running with `make` that will
+   automate all the commands, and the second one is running the test through WebUI:
+
+   1. Running the through Makefile:
+
+      -  run ``make`` in the console, will give you the list of commands
+         to run.
+
+         .. code:: shell
+
+            make
+
+      -  the output will like this:
+
+         .. code:: makefile
+
+            clean: $(suite)         Will Clean the Environment of previous runs.
+            build: $(suite)         Will Build the GeoServer Docker Image for the Environment.
+            test: $(suite)      Will running the Suite test with teamengine.
+
+      - Choose which test to run by setting the Suite environment variable:
+
+        .. warning::
+
+            The first Docker build may take a long time.
+
+        .. code:: SHELL
+
+           suite=wcs10
+
+        .. note::
+
+           Valid values for the Suite variable are
+             * wcs10
+             * wcs11
+             * wfs10
+             * wfs11
+             * wms11
+             * wms13
+
+      - Choose which GeoServer war to test by setting the ``war_url`` environment variable inside the ``Makefile``, ex:
 
-- :ref:`Teamengine Web Application<Teamengine Web Application>`
+        .. code:: C
+
+          war_url = "https://build.geoserver.org/geoserver/main/geoserver-main-latest-war.zip"
+
+      .. note::
+
+        if you don't want to do it inside the ``Makefile`` you have the option of add the variable in the command when you build the docker images.
+
+      -  To clean the local environment.
+
+         .. code:: shell
+
+            make clean suite=<suite-name>
+
+      -  To build the geoserver docker image locally.
+
+         .. code:: shell
+
+            make build suite=<suite-name>
+
+      - Alternative, with the ``war_url`` variable include:
+
+         .. code::
+
+           make build suite=<suite-name> war_url=<url-or-the-geoserver-war-file-desired>
+
+      -  To run the suite test.
+
+         .. code:: shell
+
+            make test suite=<suite-name>
+
+      -  And the last, but no less important run the full automate
+         workflow.
+
+
+
+         .. code:: shell
+
+            make clean build test suite=<suite-name>
```

Comment:
the problem is that teamengine is running with the root user so the logs are created by root.

Reply:
solved in this commit https://github.com/randomorder/geoserver/pull/3/commits/36a49e75d9c99b3948a52a63f21cd60741c65e1b

Reply:
Sudo will likely ask for password right? Best to document make seemingly ends up asking for credentials, some might get pretty spooked by such request. @randomorder is there no way to map the root user of docker to a local user, so that the files are created by the user that started the docker images?

Reply:
yes but is non trivial

typical workaround in these cases is you run the process in the docker container (that is defined at docker image build time) with a user that has the same UID of the user that is triggering the docker run (run time).

I'll double check available options with @agpenton

Reply:
If it cannot be worked around, it should be documented.